### DISCUSSION TITLE
Documentation not up to date

### DISCUSSION THREAD

**Discussion by forieux** [blackjax-devs/blackjax#511] [General]
First thanks for the toolbox.

If I look for the documentation

https://blackjax-devs.github.io/blackjax/examples/quickstart.html#nuts

it seems not up to date as the code does not run with the last release of blackjax, API has change. Moreover, the documentation is not very clear on how to use the api, I'm confused.

**Comment by twhentschel** [comment]
Are you installing from `main`? It looks like the examples should be working (I can get most of them to work, including the quickstart) when I installed from the `main` branch: see #546

**Comment by Madhav-Kanda** [comment]
You can follow the advice given by @twhentschel, or perhaps install blackjax-nightly using the command `pip install blackjax-nightly`. You'll then be able to run almost all examples!
